src/app/core/shell/sidenav/sidenav.component.html

```diff
@@ -15,16 +15,16 @@
     <div class="app-user">
       <div class="app-user-photo">
         <!-- TODO: Decision to be taken on providing this feature -->
-        <img src="assets/images/user_placeholder.png" alt="app-user-photo">
```

Comment:
undo this, restore profile image and related changes, upload feature is going to be added later, that's why i didn't ask you to change color of image.